tests/index.rs

```diff
@@ -51,6 +51,19 @@ async fn test_404() {
     }
 }
 
+#[actix_web::test]
+async fn test_404_fallback() {
+    for f in [
+        "/tests/does_not_exist.sql",
+        "/tests/does_not_exist.html",
+        "/tests/does_not_exist/",
+    ] {
+        let resp_result = req_path(f).await;
+        let resp = resp_result.unwrap();
+        assert_eq!(resp.status(), http::StatusCode::OK, "{f} isnt 200");
```

Comment:
Can we also assert that the contents match the sentence from the 404.sql file!

Reply:
Yes!